spec/helpers/orchestration_stack_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+
+describe OrchestrationStackHelper do
+  class TestClass
```

Comment:
@lpichler i am not sure if we need to create a TestClass here, can you check some of the other textual helper spec tests for reference.